src/main/java/org/gridsuite/shortcircuit/server/entities/ShortCircuitAnalysisResultEntity.java

```diff
@@ -10,26 +10,27 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
-import java.io.Serializable;
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
 import java.util.UUID;
 
 /**
- * @author Etienne Homer <etienne.homer at rte-france.com>
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
  */
 @Getter
 @AllArgsConstructor
 @NoArgsConstructor
 @Entity
-@Table(name = "result")
-public class ResultEntity implements Serializable {
+@Table(name = "shortcircuit_result")
+public class ShortCircuitAnalysisResultEntity {
 
     @Id
     private UUID resultUuid;
 
-    @Column(name = "result", columnDefinition = "CLOB")
-    private String result;
+    @Column
+    private LocalDateTime writeTimeStamp;
```

Comment:
Is there a reason to take LocalDateTime ? By experience it's better to use ZonedDateTime and specify the time zone.

Reply:
You're right, corrected.